docs/src/main/sphinx/connector/redis.rst

```diff
@@ -11,7 +11,15 @@ broken down into cells by using table definition files.
 Only Redis string and hash value types are supported; sets and zsets cannot be
 queried from Trino.
 
-The connector requires Redis 2.8.0 or later.
+Requirements
+------------
+
+Requirements for using the connector in a catalog to connect to a Redis data
+source are:
+
+* Redis 2.8.0 or higher
```

Comment:
It might be worth mentioning that Redis cluster is not supported while standard leader-follower setup is supported.

Reply:
Can you suggest a reasonable wording .. I am not familiar with Redis to know what makes sense

Reply:
@hashhar let's follow up on this. Is there an official term for "standard leader-follower setup"? Or should this just say

> Redis 2.8.0 or higher (Redis Cluster is not supported)